docs/operation/cluster_and_service_configuration.rst

```diff
@@ -0,0 +1,243 @@
+Cluster and Services Configurations
+===================================
+
+This section contains information describing the list of available Cluster and
+Services Configurations including procedures for customizing and applying any
+given Cluster and Services Configurations.
+
+Managing Cluster and Services Configurations
+********************************************
+
+Newly deployed **MetalK8s** clusters come with chosen default values for most
+Cluster services. These default values are transparent to Admin users
+and can be customized at any point in time given that Admins follow the
+documented procedure to the later.
+
+Managing Authentication
+^^^^^^^^^^^^^^^^^^^^^^^
+
+Add a local static user
+"""""""""""""""""""""""
+
+Local authentication via static users is enabled by default after a fresh
+MetalK8s installation.
+
+   .. important::
+
+      To continue using MetalK8s in cases where the external authentication
+      system fails, we advise MetalK8s administrators to leave the default
+      super admin account enabled at all times.
+
+To add a new static user, perform the following operations:
+
+#. Generate a password hash. The command below requires **apache-utils**
+   package to be installed.
+
+   .. code-block:: shell
+
+      $ htpasswd -bnBC 12 "" <replace-with-password> | tr -d ':\n'
+
+#. From the Bootstrap node, edit the ConfigMap ``metalk8s-dex-config`` and then
+   add a new entry to the already existing list using:
+
+   .. code-block:: shell
+
+      $ kubectl --kubeconfig /etc/kubernetes/admin.conf \
+         edit configmaps metalk8s-dex-config -n metalk8s-auth
+
+   The new entry should be unique and possess mandatory fields like ``email``,
+   ``hash``, ``username`` and ``userID`` like in the example below.
+
+   .. code-block:: yaml
+
+      data:
+         config.yaml: |-
+            spec:
+               localuserstore:
```

Comment:
I think you should add some `[...]` or something like that just to mention that this example do not contains the full config.yaml but only part of the field that can be edited

Reply:
ok